docs/reference/signals.md

```diff
@@ -206,3 +206,27 @@ A translatable model is a model that implements the [TranslatableMixin](wagtail.
 -   `sender` - `CopyForTranslationAction` or `CopyPageForTranslationAction`
 -   `source_obj` - The source object
 -   `target_obj` - The copy of the source object in the new locale
+
```

Comment:
We will need to add an inline ref to cross-reference (as per other comment below).

```suggestion

(init_new_page_signal)=

```